{"uuid":"dcb469cb-c629-4df8-b902-50e6e4847c05","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["ed5ef854-ee27-4810-8c0d-a66894bd2ab5"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687491676480,"stop":1687491676480},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687491676692,"stop":1687491676692},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687491676776,"stop":1687491676776},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687491676813,"stop":1687491676813},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687491676822,"stop":1687491676822}],"start":1687491676394,"stop":1687491676823}